### DISCUSSION TITLE
什么时候会出带NPU的板子？

### DISCUSSION THREAD

**Discussion by ldqmz** [AvaotaSBC/.github#3] [Q&A] [answered]
博主什么时候上架带有NPU的板子？是否有讨论群？

**Comment by SamulKyull** [comment, marked as answer]
The next batch is expected to go on sale in September with an NPU version. Currently, recognition is performed using NCNN running on the CPU. 
The QQ group number has been updated at https://github.com/orgs/AvaotaSBC/discussions/1
